Getting_Started/Vitis/example/src/u200.cfg

```diff
@@ -1,4 +1,4 @@
-platform=xilinx-u200-gen3x16-xdma-1-202110-1
+platform= xilinx_u200_gen3x16_xdma_1_202110_1
```

Comment:
@randyh62 Is this additional space necessary? The cfg content seems not changed but we're introducing diffs.

Reply:
The config is changed in that the platform used to use '-' and now uses '_', but the extra space is just a typo.